_includes/header.html

```diff
@@ -21,7 +21,7 @@
         <div class="trigger">
           {% for path in page_paths %}
             {% assign my_page = site.pages | where: "path", path | first %}
-            {% if my_page.title %}
+            {% if my_page.title and !my_page.exclude_from_nav != true %}
```

Comment:
@dirkgomez I tested and confirmed @ashawley's logic to be correct.  In the current state (8fb7b81) the boolean is working opposite from desired behavior.

Also, it is up to you if you care or not, but I noticed your first commit (41529ce2f614a540c74d1af1d216a9cfa79cea58) was done with a different e-mail address than the others, and that e-mail address is not associated with your GitHub account.  You may wish to squash/rebase (then force-push) to bring them all under one address or to add your other e-mail address to your GitHub account so it is recognized as yours.